boltz.docker-compose.yml

```
@@ -1,4 +1,36 @@
+        dockerfile: cln.Dockerfile
+      restart: unless-stopped
+      container_name: boltz-cln
+      healthcheck:
+        test: ["CMD", "lightning-cli", "--network=regtest", "getinfo"]
+        interval: 10s
+        timeout: 5s
+        retries: 10
+      command:
+        - --bind-addr=0.0.0.0:9735
+        - --bitcoin-rpcconnect=bitcoin:18443
+        - --bitcoin-rpcpassword=123 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)